### DISCUSSION TITLE
Exercise 13: atomic commit (7.2)

### DISCUSSION THREAD

**Discussion by dag7dev** [sapienzastudentsnetwork/distributed-systems#38] [Exercises]
Give an ACP that also satisfies the converse of condition AC3.

That is, if all processes vote yes, then the decision must be commit.

Why it is not a good idea to enforce this condition?

**Comment by simonesestito** [comment]
**I'm not sure at all about this solution.**

If all processes vote yes, they can still crash somehow later. But if a process crashes, the decision cannot be Yes because the crashed one won't apply the decision.

---

However, this solution is so vague for me 🤔
We may also consider logs and so on
